scripts/cnv_cromwell_tests/somatic/cnv_somatic_panel_wes_do-gc_workflow.json

```diff
@@ -1,8 +1,12 @@
 {
   "CNVSomaticPanelWorkflow.do_explicit_gc_correction": "true",
   "CNVSomaticPanelWorkflow.gatk_docker": "__GATK_DOCKER__",
+  "CNVSomaticPairWorkflow.PreprocessIntervals.bin_length": 0,
   "CNVSomaticPanelWorkflow.intervals": "/home/travis/build/broadinstitute/gatk/src/test/resources/large/cnv_somatic_workflows_test_files/ice_targets_sample-chr20.interval_list",
-  "CNVSomaticPanelWorkflow.normal_bams_list": "/home/travis/build/broadinstitute/gatk/scripts/cnv_cromwell_tests/somatic/cnv_somatic_panel_wes_input.tsv",
+  "CNVSomaticPanelWorkflow.normal_bams": ["/home/travis/build/broadinstitute/gatk/src/test/resources/large/cnv_somatic_workflows_test_files/SM-74NEG-v1-chr20-downsampled.deduplicated.bam",
```

Comment:
bais before bams to maintain sort order.

Reply:
Done